Running: (['kubectl', '--v=4', '--context', 'default-system', '--namespace', 'default', 'run', '--restart=Always', '--limits=memory=256Mi', '--requests=memory=64Mi', 'example', '--image=datawire/telepresence-k8s:0.52', '--labels=telepresence=02bb98df-4ff1-4d89-83ec-50b1a9994c5d'],)...I0718 16:54:51.098532    2624 helpers.go:193] server response object: [{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "deployments.extensions \"example\" already exists",
  "reason": "AlreadyExists",
  "details": {
    "name": "example",
    "group": "extensions",
    "kind": "deployments"
  },
  "code": 409
}]
F0718 16:54:51.098590    2624 helpers.go:114] Error from server: deployments.extensions "example" already exists
